src/app/(proper_react)/(redesign)/(authenticated)/admin/emails/actions.tsx

```diff
@@ -285,3 +291,27 @@ export async function triggerFirstDataBrokerRemovalFixed(emailAddress: string) {
     />,
   );
 }
+
+export async function triggerPlusExpirationEmail(emailAddress: string) {
+  const subscriber = await getAdminSubscriber();
+  if (!subscriber) {
+    return false;
+  }
+
+  const acceptLangHeader = await getAcceptLangHeaderInServerComponents();
+  const l10n = getL10n(acceptLangHeader);
+  await send(
+    emailAddress,
+    l10n.getString("email-plus-expiration-subject"),
+    <UpcomingExpirationEmail
+      subscriber={sanitizeSubscriberRow(subscriber)}
+      expirationDate={new Date(Date.now() + 7 * 24 * 60 * 60 * 1000)}
```

Comment:
Suggestion: We have the constant `CONST_DAY_MILLISECONDS`.

```suggestion
      expirationDate={new Date(Date.now() + 7 * CONST_DAY_MILLISECONDS)}
```

Reply:
Ah nice. 63f84e9a174b3de633107d4a3ee6c0fe46bb71ac